vtex-apps/yext-store-locator
# [YEXT-1] fixed a performance issue in individual store pages

#### What problem is this solving?

Fix loading time for store pages

#### How to test it?

Visit https://www.worldwidegolfshops.com/store/edwin-watts-golf-fl-33498/BOC
See if https://weiworkspace--sandboxusdev.myvtex.com/store/edwin-watts-golf-fl-33498/BOC loads faster
